autokeras/tasks/time_series_forecaster.py

```diff
@@ -1,7 +1,118 @@
+import pandas as pd
+
 from autokeras import auto_model
+from autokeras import hypermodels
+from autokeras import nodes as input_module
+from autokeras.tuners import greedy
+
+
+class SupervisedTimeseriesDataPipeline(auto_model.AutoModel):
```

Comment:
I don't think we need this class between AutoModel and TimeseriesForecaster. For other classes, we have such classes to avoid duplication. Since this one only has one subclass, this class is not necessary. You can just put everything in the TimeseriesForecaster.

Reply:
We can add a separate forecaster with classification head also right